packages/portal-proto/next.config.js

```diff
@@ -1,10 +1,3 @@
-// eslint-disable-next-line  @typescript-eslint/no-var-requires
-const withTM = require("next-transpile-modules")([
-  "@oncojs/survivalplot",
-  "@oncojs/react-survivalplot",
-  "@sjcrh/proteinpaint-client",
-]);
```

Comment:
I'm not entirely clear on why we needed to begin with. What changed to allow us to remove this?

Reply:
I will need @craigrbarnes' help to explain "why we needed to begin with" part, but I can tell you is that with NEXT 13 transpilePackages came [built in](https://nextjs.org/docs/app/api-reference/next-config-js/transpilePackages) as a result [next-transpile-modules](https://github.com/martpie/next-transpile-modules/releases) is not needed anymore.
Upon checking it further I noticed that transpiling was not necessary anymore as the `@oncojs/survivalplot` and `@sjcrh/proteinpaint-client` was working fine without it but I had to add `experimental: { esmExternals: true }` to make it work. Craig and I discussed about it and thought we can keep the `experimental` in the meantime.

Reply:
Nice.  Thanks for the explanation.   I've been wait for nextjs to have better es module support!